hepdata/modules/records/assets/js/hepdata_tables.js

```diff
@@ -15,7 +15,17 @@ HEPDATA.switch_table = function (listId, table_requested, table_name, status) {
   $('#' + table_requested).addClass("active");
 
 
-  HEPDATA.render_loader("#table_loader", [
+  HEPDATA.render_loader("#main_table_loader", [
+      {x: 26, y: 30, color: "#955BA5"},
+      {x: -60, y: 55, color: "#2C3E50"},
+      {x: 37, y: -10, color: "#955BA5"},
+      {x: -60, y: 10, color: "#955BA5"},
+      {x: -27, y: -30, color: "#955BA5"},
+      {x: 60, y: -55, color: "#2C3E50"}],
+    {"width": 200, "height": 200}
+  );
```

Comment:
Just a thought: if this is how it is done elsewhere in the HEPData codebase then it's fine to do it here too, but generally I would have thought applying a class and handling styling in CSS to be the better approach.

Reply:
Oh, unless this is calling a d3 function? I guess it might be difficult to do that with CSS...